proposals/pr002-hunkspace.md

```diff
@@ -0,0 +1,545 @@
+    for the publisher to support. And there will always be languages for
+    which the user community could offer a superior user
+    experience.
+
+Examples of products expected to fit this definition are: Firefox,
+Chromium, Thunderbird, Gmail.
+
+The implications for language packs are:
+
+1.  Language packs are distributed separately from the "main" body of
+    the software package.  This is done to control the software
+    distribution size.  Distribution size is a known
+    \[\[citation\_needed\]\] factor to product adoption. Developer teams
+    usually monitor it closely for this reason.
+2.  Third party language pack contributions are common and to be expected. 
+	Different approaches exist that make this possible.   For examples, see
+	Mozilla Localization or [ᎭᎴᎾ ᏗᏓᏴᎳᏛᏍᎩ ᎬᏗ Gmail ᏣᎳᎩ][gmailch0].
+
+[gmailch]: https://gmail.googleblog.com/2012/11/gmail-get-started-with-gmail-in-cherokee.html
+
+Item (2) has a consequence that a language pack for a software package may come
+from a place that the software distributor doesn't even know exists.  In the
+simplest form, you can download a language pack from a website of your choice,
+register it with your piece of software and you get a localized version of your
+program. This is approximately how [Mozilla's
+XPI](https://developer.mozilla.org/en-US/docs/Mozilla/XPI) works with
+localizations.
+
+In a world of verified execution, the XPI drop-in approach is no longer
+viable. Since only verified data can be downloaded, it is no longer
+possible for a piece of software to allow drop-in localization
+replacements. So, for a third party contribution to be usable with a
+software package, it will have to be verified at the platform level to
+ever stand a chance of appearing as a piece of software
+configuration.
+
+It would be impractical to require every piece of software to implement its own
+verification method. This means, language pack distribution and verification
+falls into the purview of the underlying software platform. In Fuchsia's case,
+that platform is the operating system.  One can imagine a centralized
+[localization] clearinghouse service, which takes care of the provision of
+localized assets, similar to how a software delivery framework (say
+[TUF](https://theupdateframework.io) provides packages and files.
+
+A data provider is a natural abstraction for a language pack.  A piece of
+software can request a hunk of data from the provider, such as a slice of CLDR.
+
+But, since we centralized the data provision, we could use exactly the
+same mechanism to request other types of localized data that is
+not a part of CLDR.  Such as, for example, a translation of the
+string "Firefox" into Urdu.  While everyone could roll their own
+approach to localization (as was the case historically) , having a
+common specification could go a long way towards a common solution that
+works well in many use cases.  Similarly to how the industry has mostly
```

Comment:
nit: change `Similarly` to `Similar` and consider merging with previous sentence.

more minor nit: you have a double space before `Similarly`